### DISCUSSION TITLE
[BUG] - The term 'Get-PnPFolderInFolder' is not recognized as the name of a cmdlet

### DISCUSSION THREAD

**Discussion by danielpcamara** [pnp/powershell#4016] [Q&A] [answered]
### Reporting an Issue or Missing Feature
I'm trying to call the cmdlet '[Get-PnPFolderInFolder](https://github.com/pnp/powershell/blob/dev/documentation/Get-PnPFolderInFolder.md)', but 'I'm getting a error saying that the cmdlet does not exist.

### Steps to reproduce behavior
Connect to a web site using ```Connect-PnPOnline -Url 'https://{domain}.sharepoint.com/sites/{sitename}'```. And then calling ```Get-PnPFolderInFolder``` in the root.

### What is the version of the Cmdlet module you are running?
I was using the 2.4.0 and return to the 2.2.0 and still get the same error. And repeat the test with 2.3, same result.
Code used to go back a version:
```
Uninstall-Module -Name "PnP.PowerShell"
Install-Module -Name "PnP.PowerShell" -MinimumVersion  '2.0.0.0' -MaximumVersion '2.2'
```
And:
```
Uninstall-Module -Name "PnP.PowerShell"
Install-Module -Name "PnP.PowerShell" -MinimumVersion  '2.0.0.0' -MaximumVersion '2.3.0.0'
```

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

Is anybody else getting the same error?

EDIT1: Others cmdlet ar working fine. But I can't use Get-PnPFolderItem since I have folders with more than 200.000 files and I run out of memory.

**Comment by veronicageek** [comment]
I can run that cmdlet in `v2.4` no problem

![image](https://github.com/pnp/powershell/assets/25181757/0eebcaaf-565b-4aa6-b984-29664d5c3d77)

**Reply by danielpcamara** [reply]
@veronicageek, any ideas why I can't call the cmdlet?

**Reply by danielpcamara** [reply]
I made another test and now I'm more confused. I download the repo to test and to my surprise using VSCode Terminal pwsh the function is working. I even double check to se if by using Win+R ```powershell``` and the error is still happening.

**Comment by danielpcamara** [comment, marked as answer]
After reviewing the documentation. I came to the conclusion that my error is due to the fact that I was running the code in ```powershell``` or ```powershell_ise```.
Just use ```pwsh``` ([download here](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.4)), this works.

**Reply by veronicageek** [reply]
@danielpcamara - We do not support PS `5.1` since version `2.x` (which needs PS `7.2` and above) for a while. If you still need to use PS5 or the ISE, then the latest version was `1.12`.
